Tensile/Source/SolutionMapper.h

```diff
@@ -194,6 +203,16 @@ class SolutionMapper : public SolutionMapperBase<ProblemDimsType>
             _exactMap.insert({pkey, solutionIdx});
         }
 
+#ifdef _WIN32
+        const DWORD nSize = _MAX_PATH;
+        static char db[nSize];
+        GetEnvironmentVariableA("TENSILE_DB", db, nSize);
+        _db = strtol(db,nullptr,0);
+
+        static char alg[nSize];
+        GetEnvironmentVariableA("TENSILE_FIND_ALG", alg, nSize); // If <0 see Algo enumeration, or >=0 specified specific solution index
+        _findAlg = strtol(alg,nullptr,0);
+#else
```

Comment:
Can you move this to a cross-platform function in Tools.h / .cpp that returns a const char*, and only pull in the windows header in the .cpp ?

Reply:
I guess but it will have to be a pointer to thread local data which shouldn't be held on to.   Should be fine for the current uses.

Reply:
Does anyone build tensile on windows ?   All of your windows support code I assumed was out of date.  I didn't really want to get into updating the old code I wasn't going to be testing locally as that is another job for someone else.   If I add to tools.cpp I guess I'll be forced to update it.   @bragadeesh do you know the history of the windows side?  Can move that discussion to an email/channel.